README.md

```diff
@@ -51,6 +51,7 @@ The belief state have three sections: semi, book and booked. Semi refers to slot
 <tr><td><a href="https://arxiv.org/abs/2005.02877">TripPy</a> (Heck et al. 2020)</td><td></td><td></td><td>55.3</td><td></td><td></td><td></td></tr>
 <tr><td><a href="https://arxiv.org/pdf/2005.00796.pdf">SimpleTOD</a> (Hosseini-Asl et al. 2020)</td><td></td><td></td><td>56.45</td><td></td><td></td><td></td></tr>
 <tr><td><a href="https://arxiv.org/pdf/2009.13570.pdf">ConvBERT-DG + Multi</a> (Mehri et al. 2020)</td><td></td><td></td><td>58.7</td><td></td><td></td><td></td></tr>
+<tr><td><a href="https://arxiv.org/pdf/2010.12850.pdf">TripPy + CoCoAug</a> (Li and  Yavuz et al. 2020)</td><td></td><td></td><td>60.53</td><td></td><td></td><td></td></tr>
```

Comment:
Trippy exists [data leakage problem](https://gitlab.cs.uni-duesseldorf.de/general/dsml/trippy-public/-/blob/master/dataset_multiwoz21.py#L446), it is not a solid work.

Reply:
> Trippy exists [data leakage problem](https://gitlab.cs.uni-duesseldorf.de/general/dsml/trippy-public/-/blob/master/dataset_multiwoz21.py#L446), it is not a solid work.

The line you refer to is for system act label, right? I think some other SOTA models also utilize additional annotation information (e.g. NADST (Le et al., 2019) uses system action and DSTQA (Zhou & Small, 2019) uses knowledge graph). Is this a problem? Thank you.

Reply:
Hi HuangLK, thanks for raising the issue. Please discuss with Shiyang and TripPy's authors about it and we'd like to be updated on the final conclusion! If there indeed exists the problem, please send a pull request to correct the number or add an annotation that these numbers are overestimated.

Reply:
> Hi HuangLK, thanks for raising the issue. Please discuss with Shiyang and TripPy's authors about it and we'd like to be updated on the final conclusion! If there indeed exists the problem, please send a pull request to correct the number or add an annotation that these numbers are overestimated.

Hi Xiaoxue, 

The link HuangLK refers to is the additional annotation from multiwoz dataset itself rather than test set leakage. See Table 1 of SimpleTOP in NeurIPS 2020 (https://arxiv.org/pdf/2005.00796.pdf). Current SOTA DST model ConvBERT-DG + Multi in the leaderboard also utilize additional open-dialogue and Dialogue GLUE dataset. Because there doesn't exist test set leakage, I think these methods are all reasonable models. Does my reply answer your questions? Thank you.

Reply:
The line I refer to means that the final informed values(from dialog actions) are checked with ground-truth values. See [the code](https://gitlab.cs.uni-duesseldorf.de/general/dsml/trippy-public/-/blob/master/dataset_multiwoz21.py#L267),the `informed_value` will be set to `none` if the  value from dialog action is not equal to ground-truth value.

Reply:
> The line I refer to means that the final informed values(from dialog actions) are checked with ground-truth values. See [the code](https://gitlab.cs.uni-duesseldorf.de/general/dsml/trippy-public/-/blob/master/dataset_multiwoz21.py#L267),the `informed_value` will be set to `none` if the value from dialog action is not equal to ground-truth value.

@HuangLK @LeeShiyang Hi, have you come to an agreement regarding `Trippy exists data leakage problem, it is not a solid work.`

Reply:
> The line I refer to means that the final informed values(from dialog actions) are checked with ground-truth values. See [the code](https://gitlab.cs.uni-duesseldorf.de/general/dsml/trippy-public/-/blob/master/dataset_multiwoz21.py#L267),the `informed_value` will be set to `none` if the value from dialog action is not equal to ground-truth value.

I think such a process is like a semantic dictionary for mapping predictions to ground-truth, which is widely used in various DST works. Actually, different works have different pre-processing/post-processing/cleaning methods, which may take effect in the final performance.

Reply:
> > The line I refer to means that the final informed values(from dialog actions) are checked with ground-truth values. See [the code](https://gitlab.cs.uni-duesseldorf.de/general/dsml/trippy-public/-/blob/master/dataset_multiwoz21.py#L267),the `informed_value` will be set to `none` if the value from dialog action is not equal to ground-truth value.
> 
> @HuangLK @LeeShiyang Hi, have you come to an agreement regarding `Trippy exists data leakage problem, it is not a solid work.`

Hi Jianguo, right now I understand what HuangLK means. Thank you.